components/mycart.css

```diff
@@ -0,0 +1,60 @@
+.cart-container {
+  margin: 5rem;
+  text-align: center;
+}
+
+.cart-heading {
+  border-bottom: 1px solid var(--secondary-color);
+  padding-bottom: 1rem;
+  margin: 2rem;
+}
+
+.cart {
+  text-decoration: none;
+  font-size: 1.5rem;
+  text-decoration: none;
+  color: var(--head-primary);
+}
+.order-detail-wrap {
+  width: 22rem;
+  height: 22rem;
```

Comment:
can use fit-content for width and height for responsiveness.

Reply:
Thank you Read new thing today